AnswerGeneratorMicroservice/main.py

```diff
@@ -0,0 +1,18 @@
+from fastapi import FastAPI, HTTPException, APIRouter
+from fastapi.responses import JSONResponse
+
+from source.schemas.input_data_schema import InputDataSchema
+from source.services.llm_answer_service import getLLMAnswer
+from source.api.llm_answer_generator_router import router as answergeneratorLLMRouter
+app = FastAPI(debug=True)
+@app.get("/")
+async def get_root():
```

Comment:
why async here ? is there a need for it ?

Reply:
no i will correct it